doc/source/dev/installing_build_dependencies.rst

```diff
@@ -30,16 +30,11 @@ Use ``pip`` to install numpy_::
 
     pip install numpy
 
-It takes some time to compile numpy, so it might be more convenient to install
-it from a binary release.
+Numpy can also be obtained via scientific python distributions such as:
 
-.. note::
-
-  Installing numpy in a virtual environment on Windows is not straightforward.
-
-  It is recommended to download a suitable binary ``.exe`` release from
-  http://www.scipy.org/Download/ and install it using ``easy_install``
-  (i.e. ``easy_install numpy-1.6.2-win32-superpack-python2.7.exe``).
+- Anaconda_
```

Comment:
The link works when I render the HTML locally.
I think backticks are only necessary if the link name contains spaces?